### PR TITLE
feat: [#188427382] Add location dependent section to landlord and short term registration tasks

### PR DESCRIPTION
**Pull request by SirSamuelJoseph**
## Description

Adds location dependent section to landlord registration tasks. Also adds tests for roadmap builder for landlord industry

### Ticket

This pull request resolves [#188427382](https://www.pivotaltracker.com/story/show/188427382).

### Steps to Test

1. Register as a Poppy landlord
2. Respond "BOTH" to the first essential question
3. Respond "NO" to the second essential question
4. Select a business structure
5. You can see the short term registration task and the landlord registration task in step 4
6. Each of them should have a location dependent section

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation on the engineering documentation website
- [x] I have not used any relative imports
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] I have checked for and removed instances of unused config from CMS
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: content/src/roadmaps/license-tasks/landlord-registration.md [hunk 1]
```diff
@@ -6,12 +6,15 @@ urlSlug: landlord-registration
...
-  property**, you must register as a landlord with your municipal clerk. You
+  property**, you must register as a landlord with your local clerk. You
   need to register each time you rent out a new property. You also need to give
   your tenant(s) a copy of the completed application form.
 industryId: residential-landlord
 webflowId: 66f3198052d588875cacb94c
 licenseCertificationClassification: undefined
+requiresLocation: true
+callToActionLink: ${municipalityWebsite}
+callToActionText: Visit the ${municipality} Website
```
Comment (seidior) [inline content/src/roadmaps/license-tasks/landlord-registration.md:17]: [dust] Idk how hard this is, but I think having this variable be `municipalityName` instead might be more descriptive and less ambiguous.

File: content/src/roadmaps/license-tasks/landlord-registration.md [hunk 2]
```diff
@@ -24,6 +27,14 @@ If you are renting out a residential property with 3 or more units, you need to
 
 Each town has its own application form and process, so check with your municipal clerk.
 
+${beginLocationDependentSection}
+
+**Check with your local clerk’s office**
+
+You can find your local clerk's office through the [${municipality} website](${municipalityWebsite}) or by contacting your [county clerk](${countyClerkWebsite}) at ${countyClerkPhone}.
```
Comment (seidior) [inline content/src/roadmaps/license-tasks/landlord-registration.md:34]: [dust] We have their county name, so should we have this be `[${countyName} county clerk]` instead?

**Review by seidior — APPROVED** [review]
Makes sense! No blockers here on my part, just some stray thoughts. Thanks, Sam.

File: content/src/roadmaps/license-tasks/landlord-registration.md (hunk 2 above)
Comment (SirSamuelJoseph) [reply content/src/roadmaps/license-tasks/landlord-registration.md:34]: I'll bring this up to content for their thoughts

File: content/src/roadmaps/license-tasks/landlord-registration.md (hunk 1 above)
Comment (SirSamuelJoseph) [reply content/src/roadmaps/license-tasks/landlord-registration.md:17]: This seems doable and makes sense, I'll see if I can make that change in this PR

**Pull request merged**